aptos-move/aptos-release-builder/src/components/consensus_config.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::utils::*;
+use anyhow::Result;
+use aptos_types::on_chain_config::OnChainConsensusConfig;
+use move_model::{code_writer::CodeWriter, emit, emitln, model::Loc};
+
+pub fn generate_consensus_upgrade_proposal(
```

Comment:
This logic is migrated from #5527. Maybe we could refactor that test later cc @igor-aptos